pkg/prepconf/prepare.go

```diff
@@ -144,5 +144,30 @@ func generateConfig(conf Config) (*visorconfig.V1, error) {
 		ServerAddr: skyenv.DefaultAppSrvAddr,
 		BinPath:    "/usr/bin/apps",
 	}
+
+	if len(bp.DMSGHTTP) > 0 {
+		var dmsghttpData dmsgHTTPServersData
+		err := json.Unmarshal([]byte(bp.DMSGHTTP), &dmsghttpData)
+		if err != nil {
+			return out, nil
+		}
+		out.Dmsg.Servers = dmsghttpData.DMSGServers
+		out.Dmsg.Discovery = dmsghttpData.DMSGDiscovery
+		out.Transport.Discovery = dmsghttpData.TransportDiscovery
+		out.Transport.AddressResolver = dmsghttpData.AddressResolver
+		out.Routing.RouteFinder = dmsghttpData.RouteFinder
+		out.UptimeTracker.Addr = dmsghttpData.UptimeTracker
+		out.Launcher.ServiceDisc = dmsghttpData.ServiceDiscovery
+	}
 	return out, nil
 }
+
+type dmsgHTTPServersData struct {
```

Comment:
Maybe we can move it to pkg, export and use [`dmsgHTTPServersData`](https://github.com/ersonp/skywire/blob/d18280346f59a769fd008057da76983896b5daaf/cmd/skywire-cli/commands/config/gen.go#L195) from `skywire` here too.